### ISSUE TITLE
There will get a warning when using 'rightButtonProps' on TextField

### ISSUE THREAD

**Issue by Macrow** [wix/react-native-ui-lib#615]
I use TextField for a search input, so I customize the right button like this:

```typescript
<TextField
  text80
  hideUnderline
  placeholder={'Search...'}
  rightButtonProps={{
    iconSource: iconSearch,
    iconColor: Colors.dark30,
    style: {position: 'absolute', right: 5, bottom: 3},
  }}
  style={{
    width: 220,
    height: 30,
    borderStyle: 'solid',
    borderColor: Colors.dark70,
    borderRadius: 15,
    backgroundColor: Colors.dark80,
    marginTop: 6,
    paddingLeft: 15,
    color: Colors.black,
  }}
/>
```
![image](https://user-images.githubusercontent.com/240792/72208625-ed313d80-34df-11ea-868b-d2242ad897dd.png)

It works fine, but I got the yellow warning:

![image](https://user-images.githubusercontent.com/240792/72208642-1782fb00-34e0-11ea-83e0-3b0a242cad43.png)

It means that I pass the wrong type props, how can I solved this ?

Thanks.

**Comment by Inbal-Tish** [comment]
@Macrow What do you have in the 'iconSearch' variable?

**Comment by Macrow** [comment]
I provide an object like this, it works, the search icon appear in the textfield.

```
{"scale": 2, "uri": "/Users/macrow/Library/Developer/CoreSimulator/Devices/BD10AEAB-049C-479D-A54E-954937A5FB2F/data/Containers/Data/Application/C4A5C5B4-83AC-488C-BB09-EB8C86862995/tmp/RNVectorIcons__Ionicons_62629_30#000000@2x.png"}
```

**Comment by Inbal-Tish** [comment]
See fix in PR #617
